websites/C/CodinGame/presence.ts

```diff
@@ -0,0 +1,205 @@
+const presence = new Presence({
+		clientId: "991078618866274364",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/ar0zfXY.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname } = document.location;
+	if (pathname === "/") presenceData.details = "Browsing...";
+	else if (pathname.includes("/home")) {
+		presenceData.details = "On the dashboard";
+		presenceData.state = `${
+			document.querySelectorAll(
+				"div[class='home-content'] > div > div > div > div > home > div > home-react > div > div > section > section > div > div > div > div"
+			)[0].textContent
+		} - Rank : ${
+			document.querySelectorAll(
+				"div[class='home-content'] > div > div > div > div > home > div > home-react > div > div > section > section > div > div > div > div"
+			)[4].textContent
+		}`;
+	} else if (pathname.includes("/training")) {
+		if (pathname.includes("/easy/")) {
+			presenceData.details = "Training: Easy";
+			presenceData.state = `${
+				document.querySelector("h1[class='title']").textContent
+			} (${document.querySelector("div[class='chart-inner']").textContent}%)`;
+		} else if (pathname.includes("/medium/")) {
+			presenceData.details = "Training: Medium";
+			presenceData.state = `${
+				document.querySelector("h1[class='title']").textContent
+			} (${document.querySelector("div[class='chart-inner']").textContent}%)`;
+		} else if (pathname.includes("/hard/")) {
+			presenceData.details = "Training: Hard";
+			presenceData.state = `${
+				document.querySelector("h1[class='title']").textContent
+			} (${document.querySelector("div[class='chart-inner']").textContent}%)`;
+		} else if (pathname.includes("/expert/")) {
+			presenceData.details = "Training: Very Hard";
```

Comment:
Consistency
```suggestion
			presenceData.details = "Training: Expert";
```

Reply:
On the website, you can see Very Hard for this category, just the path name is Expert, so I think I will leave Very hard